packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -150,21 +355,66 @@ class HttpGatewayRouteSpec extends GatewayRouteSpec {
     this.match = options.match;
   }
 
-  public bind(_scope: Construct): GatewayRouteSpecConfig {
-    const prefixPath = this.match ? this.match.prefixPath : '/';
-    if (prefixPath[0] != '/') {
-      throw new Error(`Prefix Path must start with \'/\', got: ${prefixPath}`);
+  public bind(scope: Construct): GatewayRouteSpecConfig {
+    const pathRewriteConfig = this.match?.pathRewrite?.bind(scope);
+    const pathMatchConfig = this.match?.path?.bind(scope).requestMatch;
+    const defaultTargetHostname = this.match?.rewriteRequestHostname;
+
+    // Set prefix Match to '/' if none on match properties are defined.
+    const prefixMatch = areMatchPropertiesUndefined(this.match)
```

Comment:
Does this condition still make sense, even after `rewrite` properties have been added to `match`? I don't think so?

Reply:
Rewrite is taken out from `match` so the description stays good.